### PR TITLE
Mh webren

### PR DESCRIPTION
**Pull request by mhallwac**
* new scenario tests the functionality of the default OneSearch on the
home page
* new SearchPage class tests if the OneSearch page loads fully, gets
200 status, and if the url is right.
* sleep statement is necessary due to the time it takes for the page to load
* creates the new scenario for using one of the non-default search
methods (ND Catalog)
* tests to see if the search page fully loads with the correct url and status
code
* updates the SearchPage class to accommodate non-OneSearch search methods

### CODE REVIEW

File: spec/webRenovation/functional/func_webRenovation_spec.rb [hunk 1]
```diff
@@ -79,6 +79,32 @@
     calendar = WebRenovation::Pages::CalendarPage.new
     expect(calendar).to be_on_page
   end
+
+  scenario 'Search using OneSearch from HomePage' do
+    page.driver.browser.js_errors = false
```
Comment (jeremyf) [inline spec/webRenovation/functional/func_webRenovation_spec.rb]: Not a blocker, but why are you disabling JS errors? I'd personally like them to be on.

File: spec/webRenovation/pages/search_page.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        page.has_selector?('#exlidUserAreaRibbon') &&
+        page.has_selector?('#exlidMainMenuRibbon') &&
+        page.has_selector?('.tab.onesearch.EXLSearchTabTitle.EXLSearchTabLABELOneSearch') &&
+        page.has_selector?('.tab.ndcatalog.EXLSearchTabTitle.EXLSearchTabLABELND.Campus') &&
+        page.has_selector?('#tab_art_l.tab') &&
+        page.has_selector?('#tab_jou_l.tab') &&
+        page.has_selector?('#tab_dat_l.tab') &&
+        page.has_selector?('.tab.EXLSearchTabTitle.EXLSearchTabLABELeBooks') &&
+        page.has_selector?('#goButton') &&
+        if current_url == 'http://onesearch.library.nd.edu/primo_library/libweb/action/dlSearch.do?bulkSize=10&dym=true&highlight=true&indx=1&institution=NDU&mode=Basic&onCampus=false&pcAvailabiltyMode=true&query=any%2Ccontains%2C&search_scope=nd_campus&tab=nd_campus&vid=NDU&displayField=title&displayField=creator'
```
Comment (jeremyf) [inline spec/webRenovation/pages/search_page.rb]: This URL scares me. Please make a descriptive constant that explains what this URL. (e.g. This is the principle of no magic strings; name strings that have conditional logic).

File: spec/webRenovation/functional/func_webRenovation_spec.rb (hunk 1 above)
Comment (h-parekh) [inline spec/webRenovation/functional/func_webRenovation_spec.rb]: I agree, I'll remove these on all our tests.

File: spec/webRenovation/pages/search_page.rb (hunk 2 above)
Comment (mhallwac) [inline spec/webRenovation/pages/search_page.rb]: I gave them variable names of one_search_url and nd_catalog_search_url

**Pull request merged**